src/anm/media/audio.js

```diff
@@ -47,7 +47,7 @@ var audioContext = getAudioContext();
  * @class anm.Audio
  */
 function Audio(url) {
-    this.url = url + audioExt;
+    this.url = /\.\w+$/i.test(url) ? url : url + audioExt;
```

Comment:
yes, you are right. but in cases when there is exactly one source with a known extension it is not going to work. and looks like we will have cases like that. now for this project Vova is working on